.github/workflows/ci.yml

```diff
@@ -43,28 +62,19 @@ jobs:
         restore-keys: |
           ${{ matrix.platform }}-${{ matrix.python }}-pyproject.toml-${{hashFiles('pyproject.toml')}}}
 
-    - name: Setup automation
-      run: |
-        git clone --recursive https://github.com/RedisLabsModules/readies.git
-        ./readies/bin/getpy3
 
     - name: Install Python dependencies
-      run: |
-        python3 -m pip install poetry
-        python3 -m poetry config virtualenvs.create false
-        python3 -m poetry export --dev --without-hashes -o requirements-${{matrix.platform}}-${{matrix.python}}.txt
-        python3 -m pip install -r requirements-${{matrix.platform}}-${{matrix.python}}.txt
+      run: poetry install -q
 
-    - name: Install Redis Server test dependencies
+    - name: Install Redis Server 
       run: |
-        ./readies/bin/getredis --workdir /tmp/redis -v 7 --keep
-        ln -s /tmp/redis/redis redis
-        redis-server --version
+        cd redis
+        make BUILD_TLS=yes -j 4
```

Comment:
WDYT about checking if nproc is present and running against the value? IMHO we can now not update this in the future, for future environments
```
make -C redis BUILD_TLS=yes -j `nproc` all install
```

Then you can also remove the make install call below